### PR TITLE
Add Scotto69 (PCB Edition)

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description
Adds the Scotto69 (PCB Edition) along with default keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/scottokeebs/scotto69/keymaps/default/rules.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```
Comment (zvecr) [inline keyboards/scottokeebs/scotto69/keymaps/default/rules.mk:2]:
```suggestion
ENCODER_MAP_ENABLE = yes

```

File: keyboards/scottokeebs/scotto69/rules.mk
Comment (zvecr) [inline keyboards/scottokeebs/scotto69/rules.mk:1]: Please remove this file

File: keyboards/scottokeebs/scotto69/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Scotto69 (PCB Edition)
+
+![Scotto69 (PCB Edition)](https://i.imgur.com/WNtAzcGh.jpg)
```
Comment (zvecr) [inline keyboards/scottokeebs/scotto69/readme.md:3]:
```suggestion
![Scotto69 (PCB Edition)](https://i.imgur.com/WNtAzcG.jpeg)
```

File: keyboards/scottokeebs/scotto69/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+#include QMK_KEYBOARD_H
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][1][2] = {
+    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [1] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [2] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [3] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) }
+};
```
Comment (zvecr) [inline keyboards/scottokeebs/scotto69/keymaps/default/keymap.c:26]:
```suggestion
const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
    [1] = { ENCODER_CCW_CW(_______, _______) },
    [2] = { ENCODER_CCW_CW(_______, _______) },
    [3] = { ENCODER_CCW_CW(_______, _______) }
};
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/scottokeebs/scotto69/rules.mk
Comment (joe-scotto) [reply keyboards/scottokeebs/scotto69/rules.mk:1]: Removed in [d52c002](https://github.com/qmk/qmk_firmware/pull/24402/commits/d52c002f28d25796468f1e1f95ec798af161cf8d)

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**